src/SSVOpenHexagon/Components/CWall.cpp

```diff
@@ -79,17 +85,15 @@ void CWall::moveTowardsCenter(HexagonGame& mHexagonGame,
         ssvs::moveTowards(vp, mCenterPos, speed.speed * 5.f * mFT);
     }
 
-    if(pointsOnCenter > 3 || pointsOutOfBounds > 3)
+    if(pointsOnCenter == 4 || pointsOutOfBounds == 4)
```

Comment:
Why was this changed? Maybe there could be a weird scenario where a weirdly shaped wall has its vertices counted multiple times... or maybe it can never happen...?

Reply:
I made it == 4 cause I always found it to be somewhat misleading to read > 3, which to me implied there could be polygons with an arbitrary number of vertices higher than 3.
I do not mind reverting it though.